pynamodb/models.py

```diff
@@ -881,6 +906,40 @@ def _get_save_args(self, attributes=True, null_check=True):
             kwargs[pythonic(ATTRIBUTES)] = serialized[pythonic(ATTRIBUTES)]
         return args, kwargs
 
+    def _handle_version_attribute(self, serialized_attributes, actions=None):
+        """
+        Handles modifying the request to set or increment the version attribute.
+
+        :param serialized_attributes: A dictionary mapping attribute names to serialized values.
+        :param actions: A non-empty list when performing an update, otherwise None.
+        """
+        version_condition = None
+
+        for name, attr in self.get_attributes().items():
```

Comment:
Would it be possible to keep track of the `VersionAttribute` once upfront so that we don't need to iterate through attributes each and every time?

Reply:
Yes that would make more sense. I'll pull it into an attribute where I check for duplicates in init.